docs/tutorials/Transformers_and_Tasks/csv_to_oscal_cd.md

```diff
@@ -456,3 +467,17 @@ Congratulations! You have completed this tutorial.
 
 <br>
 <br>
+-----
+
+# Examples: csv files suitable for csv-to-oscsl-cd transformation
+
+The examples given here comprise csv files that can be transformed into OSCAL Component Definitions.
+The csv files comprise required headings and values expected by the trestle task transformer, and may present optional ones as well.
+
+1. OCP4 sample input
+
+This is a simple example csv that has just one rule per control and one check per rule: [ocp4-sample-input.csv](csv_to_oscal_cd/ocp4-sample-input.csv)
+
+2. Multiple checks per rule and multiple target-components per rule: [rule-name-overlap.csv](csv_to_oscal_cd/rule-name-overlap.csv)
+
+This example shows how to specify multiple rules per control and multiple checks per rule.
```

Comment:
This shows multiple checks per rule and multiple target components have same rule id.

Reply:
Yes. And csv file had a small mix-up too. Fixed.